### DISCUSSION TITLE
翻訳状況

### DISCUSSION THREAD

**Discussion by kazupon** [vuejs-translations/docs-ja#464] [Show and tell]
翻訳対象 markdown 数: 91 file

https://github.com/vuejs-translations/docs-ja/issues/6
https://github.com/vuejs-translations/docs-ja/issues/100
https://github.com/vuejs-translations/docs-ja/issues/84

**Comment by kazupon** [comment]
## 2022/04/20
- 翻訳済: 46 files
- 翻訳率: 46 / 91 = 50%

**Reply by kazupon** [reply]
## 2022/06/29
- 翻訳済み: 52 files
- 翻訳率: 52 / 91 = about 57%

**Reply by kazupon** [reply]
Guide が残り 7 files になっている。Guides は一番良く読まれるところなので、Guides すべて翻訳されたら、一旦公開してもいい気がする。

@jay-es 
どう思いますか？

**Reply by jay-es** [reply]
@kazupon はい、問題ないと思います

**Reply by kazupon** [reply]
@jay-es 
回答、ありがとうございます。
では、Guides が一旦全部完了したら、公開しましょう。:)
